korangar/src/loaders/gamefile/mod.rs

```diff
@@ -42,6 +39,42 @@ pub struct GameFileLoader {
     archives: RwLock<Vec<Box<dyn Archive>>>,
 }
 
+impl FileLoader for GameFileLoader {
+    fn get(&self, path: &str) -> Result<Vec<u8>, FileNotFoundError> {
+        let lowercase_path = path.to_lowercase();
+        let result = self
+            .archives
+            .read()
+            .unwrap()
+            .iter()
+            .find_map(|archive| archive.get_file_by_path(&lowercase_path))
+            .ok_or(FileNotFoundError::new(path.to_owned()));
+
+        // TODO: should this be removed in the future or left in for resilience?
```

Comment:
We should only crash the application if there's no `.spr`, `.act` or `.gat` files (not sure about `.rsw` and `.gat` ones), since the official client keeps running if there's no image file.

Reply:
I know this is just a copy-paste from the other file, but just pointing it here.

Reply:
I had an issue recently with `.wav` files, they only crash if the game sound is enabled

Reply:
Can we move this issue out of the scope of this PR?